docs/admin/machine.rst

```diff
@@ -155,7 +155,7 @@ Google Translate API V3 (Advanced)
 
 Machine translation service provided by Google Cloud services.
 
-This services differs from the former one by having another mean of authentication.
+This service differs from the former one by having another mean of authentication.
```

Comment:
```suggestion
This service differs from the former one by having another means of authentication.
```
"[(functioning as singular or plural) the medium, method, or instrument used to obtain a result or achieve an enda means of communication](https://www.dictionary.com/browse/means)"

Can this second method use the same way of authentication as the first? Otherwise it is an awkward construction.